### ISSUE TITLE
fix translateSanitization

### ISSUE THREAD

**Issue by egel** [Contactis/translations-manager#7]
The error: 

```
pascalprecht.translate.$translateSanitization: No sanitization strategy has been configured. 
This can have serious security implications. 
See http://angular-translate.github.io/docs/#/guide/19_security for details.
```

On solution to complete resolve this feature is to add `LanguageService` which will manage switching between languages for user/browser/os settings.

**Comment by egel** [comment]
Added translation module (service, filter) but it waits for loopback authorization (issue #4) to add it after succes login for current session.

**Comment by egel** [comment]
Temporary bypass for constantly looped translation strings (in commit 1ab6ea8c6a10b2eb93af0c6275dc57a51005f0c3). 
In days I'll fix this, 'cause bloker on this task was simultaneously develop  authorization. When commit will be made hot fix will arrive.

**Comment by egel** [comment]
Fixed resolving local language (in commit b9aa770159060639ca213ba33dc7a9d265949f54), but need to add the project meta settings, for current supported/added project's translations to store it into database (and some other later stuff). 
It could base on already implemented Languages model where all real existed languages will be contained. Meta settings of project's languages can array of references.

**Comment by egel** [comment]
resolve #7 add all supported interface languages to object into  `LanguagesService` and resolve issue with commit 2c53c627f04b6fbcf679cc2831f3d8dcf15d5bd8
